src/api/uploads/FolderUploadNode.js

```diff
@@ -66,7 +71,12 @@ class FolderUploadNode {
         this.parentFolderId = parentFolderId;
 
         await this.createAndUploadFolder(errorCallback, isRoot);
+        if (!this.getFolderId()) {
+            // Folder was not successfully created. Do not attempt to upload its contents.
+            return;
```

Comment:
Since this is just a small function, I think we can just do below instead of a guard check

```js
// Only upload contents if container exists
if (this.getFolderId()) {
    this.addFilesToUploadQueue(this.getFormattedFiles(), noop, true);
    await this.uploadChildFolders(errorCallback);
}
        
```

Reply:
Done.